stagemonitor-web-servlet/src/main/java/org/stagemonitor/web/servlet/MonitoredHttpRequest.java

```diff
@@ -31,22 +41,15 @@
 import java.util.concurrent.RejectedExecutionException;
 import java.util.regex.Pattern;
 
-import javax.servlet.FilterChain;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpSession;
-
-import io.opentracing.Span;
-import io.opentracing.Tracer;
-import io.opentracing.propagation.Format;
-import io.opentracing.tag.Tags;
-
 public class MonitoredHttpRequest extends MonitoredRequest {
 
 	private static final Logger logger = LoggerFactory.getLogger(MonitoredHttpRequest.class);
 	public static final String CONNECTION_ID_ATTRIBUTE = "connectionId";
 	public static final String WIDGET_ALLOWED_ATTRIBUTE = "showWidgetAllowed";
 	public static final String MONITORED_HTTP_REQUEST_ATTRIBUTE = "MonitoredHttpRequest";
 	public static final String USER_AGENT_PARSED_FUTURE_ATTRIBUTE = MonitoredHttpRequest.class.getName() + ".userAgentParsedFuture";
+	public static final String GC_TIME_ATTRIBUTE = "gcTime";
```

Comment:
I like this a lot (to see GC time during request)! But I'd prefer to only have one feature per pull request. That makes it easier in case I have to revert something.